packages/fx-core/resource/package.nls.json

```diff
@@ -303,8 +303,8 @@
   "core.createProjectQuestion.projectType.officeAddin.framework.placeholder": "Select a framework",
   "core.createProjectQuestion.projectType.tab.detail": "Embed your own web content in Teams, Outlook, and the Microsoft 365 app",
   "core.createProjectQuestion.projectType.tab.title": "App Features Using a Tab",
-  "core.createProjectQuestion.projectType.copilotExtension.label": "Copilot Agent",
-  "core.createProjectQuestion.projectType.copilotExtension.title": "App Features using Copilot Agent",
+  "core.createProjectQuestion.projectType.copilotExtension.label": "Agent",
+  "core.createProjectQuestion.projectType.copilotExtension.title": "App Features using Agent",
```

Comment:
Where does this appear in the UI? It might need to be plural and lowercase, like "App Features using agents".

Reply:
@therealjohn It's the title after user chooses "Agent" capability. I think it follows the convention that each word is capitalized. ("using" should also be capitalized)
![image](https://github.com/user-attachments/assets/ebf8c775-75f5-4196-a80a-10ceec67f0aa)

Reply:
Changed to plural form: "App Features Using Agents"